infra/presubmit.py

```diff
@@ -190,12 +190,10 @@ def check_valid_emails(self):
   def check_valid_language(self):
     """Check that the language specified is valid."""
     language = self.data.get('language')
-    if not language:
-      return
-
     if language not in self.LANGUAGES_SUPPORTED:
```

Comment:
Maybe add a better message when language is None
like "Missing language attribute" ?
Also, does our documentation for C/C++ mention the language attribute in template ?

Reply:
> Also, does our documentation for C/C++ mention the language attribute in template ?

Nope, and I also need to test this before landing, if we agree that's an acceptable solution.

Reply:
done!